doc/level1.rst

```diff
@@ -54,6 +54,34 @@ will not be able to enable Autocrypt on it.
     Discuss with webmail developers how to work with, refine
     the interactions.
 
+Approach and High Level Overview
+--------------------------------
+
+The core goal of Autocrypt is to fully automate the management of both
+secret and public keys, so that users can encrypt mail without
+specialized knowledge.
+
+An Autocrypt header is introduced that transfers information about a
+sender's public keys in all their emails. Autocrypt provides a simple
```

Comment:
i prefer an "active", not passive sentence, something like "Along each sent e-mail, the Autocrypt header transfers public keys and encryption preferences"  (or some other variant)

Reply:
agreed & changed